assets/src/edit-story/components/reorderable/reorderable.js

```diff
@@ -132,17 +197,22 @@ function Reorderable({ children, onPositionChange = () => {}, ...props }) {
     state: {
       isReordering,
       currentSeparator,
+      canScrollUp,
+      canScrollDown,
     },
     actions: {
       setCurrentSeparator,
       setIsReordering,
       handleStartReordering,
+      startScroll,
     },
   };
 
   return (
     <Context.Provider value={state}>
-      <ReorderableContainer {...props}>{children}</ReorderableContainer>
+      <ReorderableContainer ref={setScrollTarget} {...props}>
```

Comment:
Can't just use the ref here, as we actually need the parent container, which is the one that actually scroll. Or we could just use the ref directly here and then use `ref.current.parentNode` where used. But ref'ing the parent directly seems more sensible here.